src/Analyzers/CSharp/Tests/CSharpAnalyzers.UnitTests.projitems

```diff
@@ -97,4 +99,7 @@
   <ItemGroup Condition="'$(DefaultLanguageSourceExtension)' != '' AND '$(BuildingInsideVisualStudio)' != 'true'">
     <ExpectedCompile Include="$(MSBuildThisFileDirectory)**\*$(DefaultLanguageSourceExtension)" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="$(MSBuildThisFileDirectory)ConvertTypeOfToNameOf\" />
+  </ItemGroup>
```

Comment:
i believ eyou should be able to remove this safely

Reply:
Agreed